src/components/Dashboard/AddressListSm.vue

```diff
@@ -1,12 +1,15 @@
 <template>
   <div id="address-list-sm">
     <template v-if="isFirm">
-        <FirmsAddressList
+      <!-- Business Address -->
+      <FirmsAddressList
         :showCompleteYourFilingMessage="showCompleteYourFilingMessage"
-        :showGrayedOut="showGrayedOut" />
+        :showGrayedOut="showGrayedOut"
+      />
     </template>
+
     <!-- when "disabled", expand all panels and disable expansion -->
-    <v-expansion-panels accordion multiple :value="expansionValue" :disabled="disabled" v-else>
+    <v-expansion-panels v-else accordion multiple :value="expansionValue" :disabled="disabled">
```

Comment:
Sometimes lint or VS Code extension says this should be first, sometimes not. But it is easier to read the code this way.